_test/test_utilities_herbert/fast_map_vs_map_performance.m

```diff
@@ -0,0 +1,141 @@
+% script to compare performance of custom fast_map class and
```

Comment:
this is not a test. It is a benchmarking script. Regardless of the need for more documentation or otherwise of tests, this is a different beast and is impenetrable without these comments.

I will mark this reveiw "requires changes" and you will reply "understood" to indicate that you have taken on board this difference

Reply:
this is test of performance, which is critical in this case so this is a kind of test.  It is not a test itself as it does not start with `test_` keyword so it will not be run automatically. 

We do not have agreement on where to place such code so I put it alongside the code which tests class validity. If there are other suggestions -- I am ready to discuss and implement them if decided. 

What should I understand and what changes I should do wrt this?